en/components/select.md

```diff
@@ -6,79 +6,77 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
-`border`
 <img class="responsive-img" src="../images/select_box.png" srcset="../images/select_box@2x.png 2x" />
-`box`
 
-## Interaction State
+## Enabled and Disabled States
 
-Each of the three Select input types above can be inserted either in an enabled or a disabled state.
+In Sketch and Adobe Xd, each of these presets can be inserted in an enabled or disabled state. In Figma on the other hand, by default the input types are always enabled and they can be switched to disable state form the property panel. 
 
 <img class="responsive-img" src="../images/select_enabledstate.png" srcset="../images/select_enabledstate@2x.png 2x" />
 <img class="responsive-img" src="../images/select_disabledstate.png" srcset="../images/select_disabledstate@2x.png 2x" />
 
-## State
+## States
+
+When the user interacts with the Select, its input goes through various selection states: **Idle** with a placeholder in the place of the content, **Filled** that shows once the user has finished adding content and moved on, **Focused** with a visual indication that the user is currently inserting a value.
 
-When the user interacts with the Select, its input goes through various selection states: **idle** with a placeholder in the place of the content, focused when the dropdown is open, filled once the user has finished with selection and moved on. These flexibility enhancements afford a more dynamic interaction design that can seamlessly flow into high-fidelity prototyping.
 
 <img class="responsive-img" src="../images/select_idle.png" srcset="../images/select_idle@2x.png 2x" />
-`idle`
 
 <img class="responsive-img" src="../images/select_focused.png" srcset="../images/select_focused@2x.png 2x" />
-`focused`
 
 <img class="responsive-img" src="../images/select_filled.png" srcset="../images/select_filled@2x.png 2x" />
-`filled`
 
 Every experienced designer uses constraints wisely to limit the user input and avoid invalid states, hence the availability of validation styles. Through the available validation styles, the Select is consistent with the standard [Input](input.md) and equipped for sophisticated designs that display success, warning, and error visuals.
 
 <img class="responsive-img" src="../images/select_success.png" srcset="../images/select_success@2x.png 2x" />
-`success`
+
 <img class="responsive-img" src="../images/select_warning.png" srcset="../images/select_warning@2x.png 2x" />
-`warning`
+
 <img class="responsive-img" src="../images/select_error.png" srcset="../images/select_error@2x.png 2x" />
-`error`
 
-In Sketch, the states and validation states are achieved with `Symbol Overrides`, while in Adobe XD we are using the `Component States` paradigm to let you easily switch between them.
+
+In Sketch, the states and validation states are achieved with `Symbol Overrides`, while in Figma and Adobe XD we are using the `Component States` paradigm to let you easily switch between them.
```

Comment:
Not sure if it's appropriate to say we use the "Component States" paradigm in Figma as well, because the approach is slightly different. I made a suggestion about this on the Combo too. I'd suggest we say something like: " In Figma you can switch between them using the "State" property from the properties panel.